app/utilities/schema.py

```diff
@@ -26,7 +25,7 @@ def load_schema_from_session_data(session_data):
 
 @cache.memoize()
 def load_schema_from_params(eq_id, form_type, language_code=DEFAULT_LANGUAGE_CODE):
-    return QuestionnaireSchema(_load_schema_file('{}_{}.json'.format(eq_id, form_type), language_code))
+    return QuestionnaireSchema(_load_schema_file('{}_{}.json'.format(eq_id, form_type), language_code), language_code)
```

Comment:
My worry here is that we're sometimes getting language_code from metadata. When/If we come to work on the card that allows for swapping of language part way through a survey we would change the session_data language_code to drive this but I wouldn't think we'd want to change the metadata. We could therefore get inconsistency.
We should reduce the times that `load_schema_from_metadata` is called to minimise this.

Reply:
I've removed most usages of `load_schema_from_metadata`. The ones that are left are:

* At initial login, as we need to get the schema before we create the session.  This is ok as at this point the language code on the metadata and session will always match.
* In `/flush` where we don't have a session